src/styles/index.scss

```diff
@@ -3 +1,53 @@
+@import './abstract/constants';
+@import './abstract/placeholders';
+@import './base/base';
+
+.calendar {
+  display: flex;
+  padding: $calendar-indent;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  @extend %calendar-width;
+
+  @each $day, $index in $days-of-week {
+    &--start-day-#{$day} :first-child {
+      margin-left: ($day-size + $gap) * $index;
+    }
+  }
```

Comment:
Instead of using `:first-child` selector, it would be more appropriate to use `:nth-child(1)`. This way, it will be more consistent with the rest of your code.